starcheck/pcad_att_check.py

```diff
@@ -0,0 +1,44 @@
+import hopper
+
+def make_pcad_attitude_check_report(backstop_file, or_list_file=None,
+                                    ofls_characteristics_file=None, out=None,
+                                    ):
+    """
+    Make a report for checking PCAD attitudes
+
+    """
+
+    # Run the commands and populate attributes in `sc`, the spacecraft state.
+    # In particular sc.checks is a dict of checks by obsid.
+    # Any state value (e.g. obsid or q_att) has a corresponding plural that
+    # gives the history of updates as a dict with a `value` and `date` key.
+    # This does not define the initial state from continuity at this time.
+    sc = hopper.run_cmds(backstop_file, or_list_file, ofls_characteristics_file,
+                         initial_state=None)
```

Comment:
Assuming a null maneuver won't work because the timing is wrong (in the other direction). E.g. the obsid won't be set by the end of the maneuver.  No worries, I'll put something into hopper to parse an attitude history file.  It's super simple.